packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -58,6 +56,14 @@ const optionalBuildDirectoryName = 'build';
 const cfnTemplateGlobPattern = '*template*.+(yaml|yml|json)';
 const parametersJson = 'parameters.json';
 
+
+const deploymentInProgressErrorMessage = (context: $TSContext) => {
+  context.print.error('A deployment in progress.');
+  context.print.error('If you believe the prior deployment was aborted, run:');
+  context.print.error('"amplify push --iterative-rollbacl" to rollback the prior deployment');
+  context.print.error('"amplify push --force" to rollback the prior deployment and re-deploy');
```

Comment:
"Otherwise, run `amplify push --force` to recover the deployment" (we don't need to disclose the fact that this includes a rollback under the hood)